src/runtime/composables/local/useAuthState.ts

```diff
@@ -44,11 +44,22 @@ export const useAuthState = (): UseAuthStateReturn => {
     rawToken
   }
 
+  onMounted(() => {
```

Comment:
Also haven't validated this one yet. I think the PR needs a good functional review in general